elastic/eui
# [EuiSuperDatePicker] Fix dates overflowing out of the delimited inputs

## Summary

Partially addresses https://github.com/elastic/kibana/issues/197892

| Before | After |
|--------|--------|
| <img src="https://raw.githubusercontent.com/elastic/eui/76ba1b0cbfbd3a2cf757729f96eca1ca10b6ea85/packages/eui/.loki/reference/chrome_mobile_Forms_EuiSuperDatePicker_EuiSuperDatePicker_Overflowing_Children.png" alt=""> | <img src="https://raw.githubusercontent.com/elastic/eui/ae891a1793b543bb8af1fd8b7f8112789635abe4/packages/eui/.loki/reference/chrome_mobile_Forms_EuiSuperDatePicker_EuiSuperDatePicker_Overflowing_Children.png" alt=""> | 

+ bonus reduction of `now` input widths, more closely matching how EuiSuperDatePicker used to behave [pre-Emotion conversion](https://eui.elastic.co/v80.0.0/#/templates/super-date-picker).

## QA

- Go to https://eui.elastic.co/pr_8109/storybook/?path=/story/forms-euisuperdatepicker-euisuperdatepicker--playground&globals=viewport:mobile1
- Click the pretty duration and then the start or end input
- [x] Switch to an Absolute date and confirm that the longer date format does not overflow outside the total input width

### General checklist

- Browser QA
    ~- [ ] Checked in both **light and dark** modes~
    ~- [ ] Checked for **accessibility** including keyboard-only and screenreader modes~
- Docs site QA - N/A
- Code quality checklist
    ~- [ ] Added or updated **[jest](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/testing/unit-testing.md) and [cypress](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/testing/cypress-testing.md) tests**~
- Release checklist
    ~- [ ] If applicable, added the **breaking change** issue label (and filled out the breaking change checklist)~
- Designer checklist - N/A
